packages/ui/src/providers/schemas.provider.tsx

```diff
@@ -2,18 +2,20 @@ import { Text, TextVariants } from '@patternfly/react-core';
 import { FunctionComponent, PropsWithChildren, createContext, useEffect, useState } from 'react';
 import { Loading } from '../components/Loading';
 import { useRuntimeContext } from '../hooks/useRuntimeContext/useRuntimeContext';
-import { CamelCatalogIndex, KaotoSchemaDefinition } from '../models';
+import { CamelCatalogIndex, KaotoSchemaDefinition, LoadingStatus } from '../models';
 import { sourceSchemaConfig } from '../models/camel';
 import { useSchemasStore } from '../store';
 import { CatalogSchemaLoader } from '../utils';
+import { LoadDefaultCatalog } from '../components/LoadDefaultCatalog';
 
 export const SchemasContext = createContext<Record<string, KaotoSchemaDefinition>>({});
 
 /**
  * Loader for the components schemas.
  */
 export const SchemasLoaderProvider: FunctionComponent<PropsWithChildren> = (props) => {
-  const [isLoading, setIsLoading] = useState(true);
+  const [loadingStatus, setLoadinStatus] = useState(LoadingStatus.Loading);
```

Comment:
typo: setLoadingStatus

Reply:
Got it, thanks for catching this up :+1: